src/plugins/subsystems/drm/drm.sh

```diff
@@ -255,8 +259,8 @@ function get_available_connectors()
       ;;
     3) # REMOTE TARGET
       local find_conn_cmd="find $SYSFS_CLASS_DRM -name 'card*'"
```

Comment:
Why not move this local to the top of the function as well?

Reply:
@rodrigosiqueira  because there is no return value to be masked, I only touched the declarations that could mask errors

Reply:
@rodrigosiqueira should I split them too?

Reply:
> @rodrigosiqueira should I split them too?

By "split them" do you mean the line that I commented? If so, I think it does not hurt to split it.